.ci/pytorch/build.sh

```diff
@@ -28,6 +28,10 @@ echo "Environment variables:"
 env
 
 if [[ "$BUILD_ENVIRONMENT" == *cuda* ]]; then
+ # Flash Attention if OOming lets try reducing the max jobs
+  # MAX_JOBS=$(nproc --ignore=4)
+  MAX_JOBS=1
```

Comment:
remove this

Reply:
Is there any connection to https://circleci.canny.io/cloud-feature-requests/p/have-nproc-accurately-reporter-number-of-cpus-available-to-container ?

Reply:
somewhat related but the number of cpus available to the container would still be too high to avoid the OOM

Reply:
you are correct, thats why in the updated code I just set the max_jobs manually based off of known amount of the container memory

Reply:
https://discuss.circleci.com/t/can-you-if-on-resource-class/40581